check-style-json.php

```diff
@@ -0,0 +1,56 @@
+<?php
+    $dir= ".";
+    $falseFiles = "";
+    $exitcode = 0;
+    $falseFiles =stylecheck($dir, $falseFiles);
```

Comment:
Bitte camelCase und ich würde $invalidFiles bevorzugen. Außerdem hätte ich gerne in der Variable ein Array der Dateien/Pfade, welche falsch sind, damit man damit besser weiterarbeiten kann.